jsapp/js/dataInterface.ts

```diff
@@ -196,24 +198,26 @@ export interface SubmissionResponse {
   __version__: string
   _attachments: SubmissionAttachment[]
   _geolocation: number[] | null[]
-  _id: number
   _notes: string[]
   _status: string
   _submission_time: string
   _submitted_by: string | null
   _tags: string[]
-  _uuid: string
   _validation_status: {
     timestamp?: number
     uid?: ValidationStatusName
     by_whom?: string
     color?: string
     label?: string
   }
-  _version_: string
+  _version_?: string
   _xform_id_string: string
   deviceid?: string
   end?: string
+  // `meta/rootUuid` is persistent across edits while `_uuid` is not;
+  // use the persistent identifier if present.
+  _id: number
+  _uuid: string
```

Comment:
Not sure how these relate to the PR either, unless I'm missing something here too 🙇‍♂️